packages/__docs__/buildScripts/samplemedia/placeholder-image.js

```diff
@@ -22,6 +22,7 @@
  * SOFTWARE.
  */
 
+// TODO raw-loader is deprecated in Webpack 5, use https://webpack.js.org/guides/asset-modules/
```

Comment:
Can you please make tickets for the TODOs you made in this ticket? At least for those that are important.

Reply:
done